apps/opik-backend/src/main/java/com/comet/opik/api/Span.java

```diff
@@ -70,4 +72,11 @@ public static class Write {
         public static class Public {
         }
     }
+
+    @JsonProperty
+    @JsonView({Span.View.Public.class})
+    @Schema(accessMode = Schema.AccessMode.READ_ONLY, description = "Duration in milliseconds as a decimal number to support sub-millisecond precision")
+    public Double duration() {
```

Comment:
It seems that we're using durations as floating number instead of using ISO 8601 format. 

Although while this is inconsistent with our timestamp formatting, which uses ISO 8601, in the rest of the service, I don't have strong feelings against it. 

However, please double check that product is fine with this and specially frontend, as they might need to convert it to a more human readable format such as ISO 8601.

I'm not sure what the FE design is going to show here.